package.json

```diff
@@ -1,155 +1,60 @@
-        "cypress": "^13.17.0",
-        "cypress-browser-permissions": "^1.1.0",
-        "cypress-real-events": "^1.13.0",
-        "cypress-recurse": "^1.35.3",
-        "cypress-vite": "^1.6.0",
-        "cypress-wait-until": "^3.0.2",
-        "dotenv": "^16.4.7",
-        "eslint": "^8.57.1",
-        "eslint-plugin-cypress": "^3.6.0",
-        "eslint-plugin-markdownlint": "^0.6.0",
-        "eslint-plugin-mocha": "^10.5.0",
-        "eslint-plugin-prettier-vue": "^5.0.0",
-        "eslint-plugin-simple-import-sort": "^12.1.1",
-        "eslint-plugin-vue": "^9.32.0",
-        "git-describe": "^4.1.1",
-        "gitconfig": "^2.0.8",
-        "googleapis": "^144.0.0",
-        "jsdom": "^26.0.0",
-        "mime-types": "^2.1.35",
-        "mocha-junit-reporter": "^2.2.1",
-        "prettier": "^3.4.2",
-        "prettier-plugin-jsdoc": "^1.3.2",
-        "rimraf": "^6.0.1",
-        "sass": "1.77.6",
-        "sharp": "^0.33.5",
-        "start-server-and-test": "^2.0.10",
-        "typescript": "~5.6.3",
-        "vite": "^5.4.11",
-        "vite-node": "^2.1.8",
-        "vite-plugin-static-copy": "^2.2.0",
-        "vitest": "^2.1.8",
-        "vue-tsc": "^2.2.0",
-        "write-yaml-file": "^5.0.0",
-        "yargs": "^17.7.2"
-    },
-    "engines": {
-        "node": ">=22",
-        "npm": "10.*.*"
-    },
-    "type": "module",
-    "engineStrict": true
+  "name": "web-mapviewer-monorepo",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "dev": "npm run dev --workspace=web-mapviewer",
+    "build": "npm run build --workspaces",
+    "build:dev": "npm run build:dev --workspaces",
+    "build:int": "npm run build:int --workspaces",
+    "build:prod": "npm run build:prod --workspaces",
+    "preview": "npm run build --workspace=geoadmin && npm run preview --workspace=web-mapviewer",
+    "preview:dev": "npm run build:dev --workspace=geoadmin && npm run preview:dev --workspace=web-mapviewer",
+    "preview:int": "npm run build:int --workspace=geoadmin && npm run preview:int --workspace=web-mapviewer",
+    "preview:prod": "npm run build:prod --workspace=geoadmin && npm run preview:prod --workspace=web-mapviewer",
+    "test:unit": "npm run test:unit --workspaces --if-present",
+    "test:unit:watch": "npm run test:unit:watch --workspaces --if-present",
+    "test:e2e": "npm run test:e2e --workspace=web-mapviewer",
+    "test:e2e:ci": "npm run test:e2e:ci --workspace=web-mapviewer",
+    "test:e2e:headless": "npm run test:e2e:headless --workspace=web-mapviewer",
+    "test:component": "npm run test:component --workspace=web-mapviewer",
+    "test:component:ci": "npm run test:component:ci --workspace=web-mapviewer",
+    "lint": "eslint --fix",
+    "lint:no-fix": "eslint"
+  },
+  "engines": {
+    "node": ">=22",
+    "npm": "10.*.*"
+  },
+  "dependencies": {
+    "@turf/turf": "^7.2.0",
+    "cypress": "^14.0.1",
+    "vue": "^3.5.13",
+    "vue-i18n": "^11.1.0"
+  },
+  "devDependencies": {
```

Comment:
Is all this still needed on this level?

Reply:
I wanted to make sure we are using the same Vue and Vue-i18n version across all packages (I'll start exporting Vue component in the next PR)